gradle.properties

```diff
@@ -1,4 +1,4 @@
-version = 9.2.0-SNAPSHOT
+version = 9.2.1.001-SNAPSHOT
```

Comment:
Leave version as is please and let maintainers handle proper versioning.

Reply:
Reverted. I did this to assemble in my company's project, and versions were all conflicting.